lib/chewy/type/mapping.rb

```diff
@@ -33,9 +33,11 @@ module ClassMethods
         #     end
         #   end
         #
-        def root(options = {}, &block)
-          raise 'Root is already defined' if root_object
-          build_root(options, &block)
+        def root(options = {})
```

Comment:
You are not supporting Ruby < 2.0, consider using `**options` instead. You will gain that keywords arguments splat accepts only hashes with symbols as keys, no need to convert them down the road.